server/main.py

```diff
@@ -57,23 +44,32 @@ def get_application() -> FastAPI:
         allow_methods=["*"],
         allow_headers=["*"],
     )
-    application.add_event_handler("startup", create_start_app_handler(application))
-    application.add_event_handler("shutdown", create_stop_app_handler(application))
 
     return application
 
 
 app = get_application()
 
+# TODO: move to API folder
+@app.get("/api/v1/coastlines")
+async def get_coastlines(
+    db_session: AsyncSession = Depends(get_db),
+) -> models.Coastline:
+
+    # `selectinload`: alternative approach to `joinedload`
+    stmt = (
+        select(models.Coastline)
+        .options(joinedload(models.Coastline.buoy, innerjoin=True))
```

Comment:
Which table are you doing an inner join with? I thought Coastline table is already mapped with linestrings of ~1 mile long coast lines and buoys (i.e. our 44065 buoy and rockaway mile segments)

Reply:
We're querying "coastlines" and joining "buoys" by station_id.

Every row in coastlines will contain station_id (this is called a foreign key).